### ISSUE TITLE
Tutorial mode

### ISSUE THREAD

**Issue by czepluch** [raiden-network/webui#344]
### Problem
It can be hard to grasp how Raiden works and why it's awesome. In Raiden and the webui we have a lot of language and terminology that is hard to understand for the layman. I think especially the first time people open the webui, they have a very hard time grasping the concepts.

### Proposed solution
Make a "tutorial" overlay to users the first time they go to the webui. This tutorial should help them understand functionality and terminology of the webui and explain in layman terms how it's an improvement over current Ethereum user experiences. 

### More ideas / alternative solution
As an alternative or an extension in order for users to remember what things are, we could add small "on hover" tool tips to all terms to make it easy to remember / understand what things are.

**Comment by christianbrb** [comment]
Thanks for opening the issue :) For the old WebUI the explanation of the terms and concepts has been included within the docs
- Example: https://docs.raiden.network/using-raiden/the-raiden-web-interface/join-a-token-network

CC @taleldayekh

**Comment by czepluch** [comment]
Yes, but a first time user can't be bothered having to look at the docs to figure out what things are. And by user here, I mean someone who is using the webui to make transfers. Not someone who is using Raiden by implementing it into their own dapp.

**Comment by manuelwedler** [comment]
A tutorial mode would make things so much easier for first time users. Thanks for opening the issue! Nonetheless the docs should be updated.

**Comment by kelsos** [comment]
While this is for vue, some similar functionality like that would be nice for the first run https://pulsardev.github.io/vue-tour/

**Comment by manuelwedler** [comment]
There is a similar library for Angular: https://tnicola.github.io/ngx-joyride/home

**Comment by czepluch** [comment]
Some inspiration can be found in how Erdstall guides the user through their demo: https://erdstall.dev/index.php/demo/
